test/src/kbasesearchengine/test/integration/SearchAPIIntegrationTest.java

```diff
@@ -419,6 +530,231 @@ public void narrativeDecoration() throws Exception {
         NarrativeInfoDecoratorTest.compare(res.getAccessGroupNarrativeInfo(), expected);
     }
 
+    @Test
+    public void getObjectInfo3() throws Exception {
```

Comment:
This test should be move to WorkspaceEventHandlerTest.java right?

Reply:
There is already a getObjectInfo3() test in WorkspaceEventHandlerTest. I added another one in this integration test, which will actually make use of the getObjectInfo3() and verify the results.

Reply:
Then I suggest removing SearchAPIIntegrationTest.getObjectInfo3() because it is a duplication of the WorkspaceEventHandler.getObjectInfo3() test. Also this test does not belong in Search API integration testing. If you think it is necessary to have this test because it is an integration test, then create a separate WorkspaceEventHandlerIntegrationTest class and place this test there.